### ISSUE TITLE
cli fails when a corrupt repo exists

### ISSUE THREAD

**Issue by G-Rath** [getsentry/sentry-cli#122]
This is a case which shouldn't really be possible, but has managed to happen to me:

Somehow we now have in our Sentry a repo marked "organization-75246 - Unknown Provider".

When running the sentry-cli (specifically `sentry-cli releases set-commits --auto $VERSION`), I get this error: `error: bad json: invalid type: null, expected a string at line 1 column 64`.

This is because that rogue repo has a url value of `null`. 

I'm unable to delete the repo from Sentry, being told "An error has occurred" - I've sent an email to Sentry, who will hopefully get it resolved quickly.

While strictly speaking this is a bug in Sentry, not the cli, I feel that the cli could handle it a bit better, by giving us an option to have that repo ignored (since this is a show stopper for me, that's out of my hands), or at least having a better error message, since "bad json" confused me until I figured I could use `--log-level=debug` and saw the JSON for myself.

**Comment by mitsuhiko** [comment]
I think this is a server error that we can't do anything in the cli about. Closing for now.

**Comment by G-Rath** [comment]
@mitsuhiko While yes I agree it's a server issue, I was more hoping for a possible work around, since as stated the issue completely prevents me from using the Sentry-CLI.

**Comment by mitsuhiko** [comment]
@G-Rath in case this is on sentry.io please reach out to customer support. If this is on self hosted sentry please shoot me a mail on armin@sentry.io how to fix this.

**Comment by G-Rath** [comment]
@mitsuhiko Thanks for the offer and quick-reply, but customer support have already resolved this (about 2 days after I reported this issue).

I understand the closing, and that this is a rare occurrence. I wanted to point out the fact that if we started using this cli in a production-level environment, and this happened again, it would *every* instance of automation that used this too.

A simple null check should prevent this from being a complete show-stopped.
